### DISCUSSION TITLE
Question for Em or BinBash

### DISCUSSION THREAD

**Discussion by the-skiddo** [QuiteAFancyEmerald/InvisiProxy#147] [General]
Just curious, do you have a copy of a school monitoring/blocking extension (e.g- Securly) to test and tell when and if a link is blocked, or do you rely on what other users tell you? Thanks!

**Comment by QuiteAFancyEmerald** [comment]
Yeah my district actually has Securly so for that particular filter its pretty easy to test sites. When it comes to figuring out whether or not a site is blocked globally I will have others with the same filter test it. For any other filters I also depend on the latter.

**Reply by the-skiddo** [reply]
Ah, I see. Thats pretty neat!
